agenta-web/src/components/TestSetTable/InsertTestSetNameModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import {Modal} from "antd"
+
+interface Props {
+    isModalOpen: boolean
+    setIsModalOpen: (value: boolean) => void
+}
+
+const TestSetMusHaveNameModal: React.FC<Props> = ({
+    isModalOpen,
+    setIsModalOpen,
+}) => {
+    const handleCloseModal = () => setIsModalOpen(false)
+
+    const handleDismiss = () => {
+        handleCloseModal()
+    }
+
+    return (
+        <Modal
```

Comment:
Please use something to show warning or error icon instead. ant.design have some options for that
<img width="931" alt="Screenshot 2023-07-27 at 11 25 17" src="https://github.com/Agenta-AI/agenta/assets/4510758/348b1ed6-5fa6-43fc-b9a7-9c26cd182436">

Reply:
Noted!